week-4/E-arrays-of-objects/exercise-3.js

```diff
@@ -59,16 +59,28 @@ var restaurantFinderApplication = {
   applicationVersion: "1.0",
   restaurants: restaurants,
   findAvailableRestaurants: function(numberOfPeople) {
+    return restaurants
+      .filter(
+        number => number.totalSeats - number.numberOfCustomers >= numberOfPeople
+      )
+      .map(function(names) {
+        return names.name;
+      });
+
     // Complete here
   },
   findRestaurantServingDish: function(dishName) {
-    // Complete here
+    return restaurants
+      .filter(dish => dish.menu.includes(dishName))
```

Comment:
Similar thing here. What kind of object is `dish` ? What would be a better name for the parameter?